widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizer.kt

```diff
@@ -68,11 +67,7 @@ class ActivityWatcherForCallVisualizer(
 
     override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {}
 
-    override fun onActivityDestroyed(activity: Activity) {
-        if (activity.isFinishing) {
-            controller.onActivityDestroyed()
-        }
-    }
+    override fun onActivityDestroyed(activity: Activity) {}
```

Comment:
I would like to offer some basic `SimpleActivityWathcer` class, which will override all the required methods, and then inherit it, in the final class then we will have only those methods that we need.

e.g.
```kotlin
package com.glia.exampleapp

import android.app.Activity
import android.app.Application.ActivityLifecycleCallbacks
import android.os.Bundle

internal open class SimpleActivityWatcher : ActivityLifecycleCallbacks {
    override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {}
    override fun onActivityStarted(activity: Activity) {}
    override fun onActivityResumed(activity: Activity) {}
    override fun onActivityPaused(activity: Activity) {}
    override fun onActivityStopped(activity: Activity) {}
    override fun onActivitySaveInstanceState(activity: Activity, outState: Bundle) {}
    override fun onActivityDestroyed(activity: Activity) {}
}

internal class FinalWatcher : SimpleActivityWatcher() {
    override fun onActivityStarted(activity: Activity) {

    }
}
```

Reply:
Good point, thank you.
Made this change as long as several others, including making classes `internal`.
Please have a look, when you have time.